opentreemap/geocode/tests.py

```diff
@@ -22,40 +27,57 @@ def setUp(self):
         self.factory = RequestFactory()
 
     def test_azavea_office_geocodes_correctly(self):
-        res = geocode(MockGeocodeRequest('340 n 12th st philadelphia'))
+        extent = {
+            'xmin': -8475485,
+            'xmax': -8280250,
+            'ymin': 4643135,
+            'ymax': 4954810,
+        }
+
+        # Fetch suggestions from ESRI suggestion engine (as the front end does)
+        url = (
+            'https://geocode.arcgis.com/arcgis/rest/services/'
+            'World/GeocodeServer/suggest?' +
+            urlencode({
+                'f': 'json',
+                'searchExtent': json.dumps(dict({
+                    'spatialReference': {'wkid': 102100}
+                }, **extent)),
+                'text': '340 n 12th st philadelphia'
+            }))
+        result = json.loads(requests.get(url).content)
```

Comment:
Seems like a tradeoff. With your suggestion the test would run a bit faster, but is vulnerable to changes in the ESRI service. As-is, if ESRI changes their magic key generation formula the test won't break.